### DISCUSSION TITLE
Timelord on separate machine

### DISCUSSION THREAD

**Discussion by isipisi89** [Chia-Network/chia-blockchain#6753] [Q&A] [answered]
Can someone please write an tutorial how to run a timelord on a separate machine and connect with an full node?
I cant find any info about this.

**Comment by loppefaaret** [comment, marked as answer]
you would need to do the "copy CA from one machine, and do a `chia init -c ca/` on the other machine", to base both set of ssl keys on the same CA - once that is done, you woud edit the `config.yaml` file, under the "timelord:" section, the "full_node_peer: 10.1.1.31" is to be set to the IP or hostname of the full_node serving the timelord

**Reply by isipisi89** [reply]
have i need to edit the config on my full node too? or just the config on the timelord?

**Reply by loppefaaret** [reply]
just the timelord config is needed to be set

**Reply by isipisi89** [reply]
> just the timelord config is needed to be set

ok now if i want to launch the timelord on the separate machine do i use:

chia start timelord-only

or 

chia start timelord-launcher- only

**Reply by loppefaaret** [reply]
both - i believe

**Reply by isipisi89** [reply]
yes you are right. its working now THX!

**Comment by isipisi89** [comment]
So my timelord is sending some proofs. Dose this mean that my timelod has won the race?

VDF Client: Sending proof
VDF Client: Sent proof
VDF Client: Running proving for iter: 5867938
Got 2-wesolowski proof for iteration: 4123042.
Proof: 00009c5e9046699ebd5ca5a62d7d2f5d37425da4ab0c6c1811ca4ff335c5d7ca433f079abbb1ed4703dae54f693b386f653f353587a10bfb9b2be179a92b3010570afdce4719417bee557ce0e714c1479c1bc32b99f30a8a4c911aa7f35b0d223d08020000003098fb91c3500439f8bfbe381489a064cebbb58fe5cf798cfa972f3aed6295f66a5898c1f0c8d7f548c3119ee7faf9375d85e5d8aba16830301ccb395efc93038b6ba84a4b74afccfe42fe2956843d1e7a70a7485fc2f7c28b21a1eeacaf4805040200000000000dfae8e57a6a3f164989d892ea6c7ec2a649ef86d6af01cc6df2dab4d806c0de2e74359d0200f87114a7505fc6dc2d909a50ceb0a68f847f7f3572d399819e729cab491af42946977a750fd35fd2198d6e20635f1b8a5d901a4369e4ab70a61297abefeba05e3fd0fa6b929964fd0946ace06a75b20a867b10ef34361d5dde38f5102180823d0100000000000029f0b8a91fcceb90de9d3a9556c78f31819dcd14948a9be078f14a9b43a6605a9c6012930300d9153bebef9b3b8bbdfd9fdd7b2ce81bbd02036448fdd8618ea3dc4b23a255fde10d3af89496a860298469be7626e0fe99a100a6bb8d508c31f6af82cf399921d26a2762af5dde5750b0a6e549d791ab8e4d66093eba5af6564a27c5aaa3de150100
VDF Client: Sending proof
VDF Client: Sent proof
VDF Client: Running proving for iter: 7612834
Got 2-wesolowski proof for iteration: 5867938.
Proof: 00004e1b0188b2574ebc5a7c0fcd433427f03ee2af46c58ed454fb77d29adc999cadf91dade8b7a888b1d1c123ee06562d7c04bc8d3f6d488134aa76431d93e0f80425b49cf9ad379ca0b11ff3f8995a0245e49cbfbc36bb97b18bf2a7f8d65c3c0a020101000c3d6b14eb1abba2b60213eb4598277a6e3c71d99a8069d7d76c7feb9b2abe119e76fd7fb6660293ec886493dc0b3c1c6becccf2f3c25348e7e15f9d22780358abd469d8ae7cfd7067de2ebe7366536bfb37d774efb7fd3c6d94666d76380a2e0100000000000013e5c08db758743732fcdaf54e6254b4c758eee6da22d8e04ba81694cccffeb2d05c8a5503008fc69f7030e7ab1253b576be11ddf42a3e37f5dd7176c033cc0ffa6d063eab1d8199111a6303fcd268e5f1cbc1a1af9451818921616be16a9c3cd27a745ed402a18fe42027b8bd95c57ea135b2c3832e34683463a13c285bc670503620160106020000000000003bb0dcdfedf0aeb6caf53ebea18cc76b5494c551206d48d94a9798a7a37694a3caf2893b020025afaf0513b928c4d96eae92af490c9fd62521741579247148c3794837661f0b2710338d2f600b4daacbddb459b76954550e952f3aae9fee2397f447850bae2a48c6eaf80f68311bea69a12d9b743cf13bdf60c07af4bd80b4cff35e921a48090100

**Reply by loppefaaret** [reply]
it answered a proof with a vdf, yes!
but if there isn't another timelord already done 🤷 
you are basically doing the same work, you just haven't seen the work completed by another timelord yet, so you process it
